contentcuration/contentcuration/templates/clipboard.html

```diff
@@ -58,80 +145,77 @@
 					}
 				</script>
 			</select>
-			<a href="#add_step_2"><button  type="submit" id="explore">Explore this Channel</button></a>
-		</div>
-	</div>
-	
-	<div id="add_step_2">
-		<h4>Adding Content To Topic</h4>
-		<a href="#clipboard_overview" id="back">X</a>
-		<nav>
-			<a href="#add_step_1">1 source</a> -------- <b>2 choose</b> -------- <a href="#add_step_3">3 metadata</a></nav>
-		
-		<div id="choose_content">
-			<h6>2. Adding content from another channel</h6>
-			
-			<div id = "channel_content_area">
-				<h5><span><</span>Channel</h5>
+			<div id = "files_content_area" class="container list_content">
+				<h5>Path: Math ></h5>
 				<ul>
 					<script type="text/javascript">
-						var channels = ["Math", "English"];
-						for (var i = 0; i < channels.length; i++) {
-							document.write("<li><input type=\"checkbox\">&nbsp" + channels[i] + "</li>");
+						for (var i = 0; i < 5; i++) {
+							document.write("<li><span class=\"pull-left\">+</span>Folder " + i + "<span class=\"pull-right\">></span></li>");
+						}
+						for (var i = 0; i < 10; i++) {
+							document.write("<li><a href=\"#pdf\"><img src=\"{% static 'img/preview.png' %}\" alt=\"preview\"/></a>File " + i + "</li>");
 						}
 					</script>
 				</ul>
 			</div>
-			
-			<div id = "selected_content_area">
-				<h5>Files to be added:</h5>
+			<div id = "selected_content_area" class="container list_content">
+				<h5>Files or folders to be added:</h5>
 				<ul>
 					<script type="text/javascript">
-						var channels = ["Content Item", "Content Item","Content Item", "Content Item"];
-						for (var i = 0; i < 50; i++) {
-							document.write("<li>Content " + i + "</li>");
+						for (var i = 0; i < 5; i++) {
+							document.write("<li><img src=\"\" alt=\"folder\"/>Folder " + i + "<span class=\"pull-right\">X</span></li>");
+						}
+						for (var i = 0; i < 10; i++) {
+							document.write("<li><a href=\"#pdf\"><img src=\"{% static 'img/preview.png' %}\" alt=\"preview\"/></a>File " + i + "<span class=\"pull-right\">X</span></li>");
 						}
 					</script>
 				</ul>
 			</div>
-
-			<a href="#add_step_3"><button  type="submit" id="explore">Continue to step 3</button></a>
+			<a href="#clipboard_new_content" class="btn pull-left btn-link">Previous</a>
+			<a href="#clipboard_metadata" class="btn pull-right btn-link">Next</a>
 		</div>
 	</div>
-	
-	<div id="add_step_3">
-		<h4>Adding Content To Topic</h4>
-		<a href="#clipboard_overview" id="back">X</a>
-		<nav>
-			<a href="#add_step_1">1 source</a> -------- <a href="#add_step_2">2 choose</a> -------- <b>3 metadata</b>
-		</nav>
-		
-		<div id="meta_content">
-			<h6>3. Review metadata</h6>
-
-			<div id = "meta_list">
+	<div id="clipboard_metadata">
+		<div class="clipboard_header">
+		<h3 class="text-center">
+			<img src="{% static 'img/' %}" alt="add" class="pull-left"/>
+			Adding Content to Topic
+			<a href="#" class="back pull-right btn-link">X</a>
+		</h3>
+			<table>
```

Comment:
may need to figure out how to restructure this progress bar. tables should not be used unless there is data to handle and display